code/obj/item/clothing/fruithat.dm

```diff
@@ -19,8 +19,6 @@
 				M.nutrition += 20
 				playsound(M.loc,"sound/items/eatfood.ogg", rand(10,50), 1)
 				if (!src.bites)
```

Comment:
i killed every instance of "finishes eating" in the code because there's already chat messages that indicate when people take bites from things. if you were to eat a bunch of small items (like cake slices) consecutively, chat would be spammed with two messages per slice with no message stacking. this condenses the 20 chat messages from eating an entire cake into one stacked message x10. i modified the other instances as well for consistency, i could always revert them if you don't like the idea for everything.